scripts/generate-release-changelog-links.hs

```diff
@@ -298,25 +291,33 @@ runGitHub :: GitHub.GitHubRW req res => GitHubAccessToken -> req -> res
 runGitHub (GitHubAccessToken tok) =
     GitHub.github (GitHub.OAuth tok)
 
-generatePandoc :: [(PkgName, Ver, Maybe (Text, Text))] -> Pandoc.Pandoc
-generatePandoc ps =
-  Pandoc.Pandoc mempty
-    [ Pandoc.Plain [Pandoc.Str "Package changelogs"]
-    , Pandoc.Table mempty (Pandoc.Caption Nothing []) colSpec tableHead [tableBody] (Pandoc.TableFoot mempty mempty)
-    ]
+generateMarkdown :: [(PkgName, Ver, Maybe (Text, Text))] -> Text
+generateMarkdown changelogPaths =
+  let
+    rows  = mkHeader : map mkRow changelogPaths
+    table = render rows
+  in Text.unlines $ "Package changelogs" : "" : table
   where
-    colSpec = replicate 3 (Pandoc.AlignDefault, Pandoc.ColWidthDefault)
-    tableHead = Pandoc.TableHead mempty [Pandoc.Row mempty tableHeadCells]
-    tableHeadCells =
-      [ mkCell [Pandoc.Str "Package"]
-      , mkCell [Pandoc.Str "Version"]
-      , mkCell [Pandoc.Str "Changelog"]
-      ]
-    tableBody = Pandoc.TableBody mempty 0 [] (fmap mkTableRow ps)
-    mkTableRow (PkgName n, v, linkMaybe) =
-      Pandoc.Row mempty
-        [ mkCell [Pandoc.Str n]
-        , mkCell [Pandoc.Str (dispVer v)]
-        , mkCell (foldMap (\(fn, link) -> [Pandoc.Link mempty [Pandoc.Str fn] (link, fn)]) linkMaybe)
-        ]
-    mkCell t = Pandoc.Cell mempty Pandoc.AlignDefault 1 1 [Pandoc.Plain t]
+    mkHeader                        = ["Package", "Version", "Changelog"]
+    mkRow (PkgName n, v, linkMaybe) = [n        , dispVer v, dispLink linkMaybe]
+
+    -- example result: [CHANGELOG.md](https://github.com/IntersectMBO/cardano-base/blob/f11ddc7f/cardano-slotting/CHANGELOG.md "CHANGELOG.md")
+    dispLink (Just (file, link)) = mconcat ["[", file, "](", link, " \"", file, "\")"]
```

Comment:
I think using `format` would look better here, especially since the rest of the file uses it a lot. Perhaps:

```suggestion
    dispLink (Just (file, link)) = format $ "["%file%"]("%link%" \""%file%"\")"
```

However, if you prefer the `mconcat` version I think that's fine too.

Reply:
Great suggestion - adopted it.